### DISCUSSION TITLE
8.7.0

### DISCUSSION THREAD

**Discussion by wasabii** [ikvmnet/ikvm#436] [Announcements]
<hr /><em>This discussion was created from the release <a href='https://github.com/ikvmnet/ikvm/releases/tag/8.7.0'>8.7.0</a>.</em>

**Comment by BiologyTools** [comment]
This is huge thank you @wasabi and other developers. I know everyone using microscopes will love having Bioformats package working in .NET6 on all platforms.  :rocket:
